### ISSUE TITLE
Sidebar Swipe Function

### ISSUE THREAD

**Issue by sirius16** [clipto-pro/Android#18]
On the app 
Free Multi Clipboard Manager
https://play.google.com/store/apps/details?id=com.cv.copybubble
There was a sidebar swipe that allowed users to access the clipboard from a bubble at the bottom

<img src="https://user-images.githubusercontent.com/4426180/73614711-86c7a880-45cf-11ea-99b4-0fa3be2e22bb.png" alt="Screenshot_20200202_143246_temp" style="box-sizing: initial; border-style: none; max-width: 100%;" height=500>

And I was wondering if it would be possible to do that for Clipto Pro? Because, the app notification gets lost and it's very inconvenient to have to keep scrolling through numerous notifications, whereas the swipe gesture allows for the user to pull up a hovering pop up (draw over app)

<img src="https://user-images.githubusercontent.com/4426180/73614853-fb4f1700-45d0-11ea-8913-dd5cbf4b9034.jpg" alt="Screenshot_20200202-153006" style="box-sizing: initial; border-style: none; max-width: 100%;" height=500>

so you can select the clip, copy it or whatever, and get back to what was being done previously, was wondering if the same could be done for this app

**Comment by atrashler** [comment]
I will check it. Probably the bubble is not the best option (very specific on every platform and does not cover some specific devices) and it should exist some more optimal solution.. But need to check first.
Thanks!

**Comment by d0x360** [comment]
A bubble wouldn't work on Samsung phones because at the bottom we have a bubble for Samsung pay and on the right we have another for slide out windows.

I know I've brought it up before but a moveable popup window that could be summoned via notification would be great.

Also for OP if your notification is going away make sure your system isn't putting the app to sleep. Unless I'm misunderstanding what you're saying.  I've never had the notification go away on Android 8, 9 or 10 and I've been using the app since about about a month after release

**Comment by sirius16** [comment]
> Also for OP if your notification is going away make sure your system isn't putting the app to sleep. Unless I'm misunderstanding what you're saying. I've never had the notification go away on Android 8, 9 or 10 and I've been using the app since about about a month after release

Nah, my problem isn't that the notification goes away, that's fine, my problem is that it's often buried deep in my notifications, which means that every time I want to open the clipboard, I have to scroll through a bunch of notifications (and the location changes every time). Point of the bottom bubble (or floating bubble, that's fine to) would be to make access easier, one touch

**Comment by d0x360** [comment]
Weird, mines always the first notification no matter what.

**Comment by sirius16** [comment]
Really? What kind of phone do you have?

**Comment by d0x360** [comment]
I have a Samsung Galaxy Note 10+ running Android 10 with the Feb update but
the notification stayed on top on my note 9 on Android 8 & 9 as well.  I
think I also tested it on my Galaxy tab, Nexus 6, and HTC One m8.  I was
really hammering away at the app looking for bugs on multiple devices.  Now
I just use the note 10+, Galaxy tab and Windows 10.

As for swipe function ideas...

I actually configured a button called the bixby button (Samsung's garbage
version of Google assistant) into a button that opens clipto pro. More
companies should put a hardware button that can be configured to open
specific apps.

You could also always use a different launcher (I use Nova but there are
many) that supports gestures so you could open the app that way.

A bubble or side swipe function isn't a bad idea but it would need to have
an option to disable it because some phones (like mine) already have
multiple swipe zones for other functions that would get in the way.
